libs/ui-lib/lib/ocm/components/featureSupportLevels/FeatureSupportLevelProvider.tsx

```diff
@@ -123,80 +70,81 @@ export const FeatureSupportLevelProvider: React.FC<SupportLevelProviderProps> =
     [cluster?.cpuArchitecture, infraEnv?.cpuArchitecture, infraEnv?.staticNetworkConfig],
   );
 
-  const getVersionSupportLevelsMapCallback = React.useCallback(
-    (versionName: string): FeatureIdToSupportLevel | undefined => {
-      return getVersionSupportLevelsMap(versionName, supportLevelData);
-    },
-    [supportLevelData],
-  );
+  const getFeatureSupportLevels = React.useCallback((): NewFeatureSupportLevelMap => {
+    return supportLevelData;
+  }, [supportLevelData]);
 
   const getFeatureSupportLevel = React.useCallback(
-    (versionName: string, featureId: FeatureId): SupportLevel | undefined => {
-      const versionSupportLevelData = getVersionSupportLevelsMapCallback(versionName);
-      return versionSupportLevelData ? versionSupportLevelData[featureId] : undefined;
+    (
+      featureId: FeatureId,
+      supportLevelDataNew?: NewFeatureSupportLevelMap,
+    ): SupportLevel | undefined => {
       // eslint-disable-next-line react-hooks/exhaustive-deps
+      if (supportLevelDataNew) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        return supportLevelDataNew ? (supportLevelDataNew[featureId] as SupportLevel) : undefined;
```

Comment:
With the "if", you already know `supportLevelDataNew` is defined. Same for the "else".

Reply:
supportLevelDataNew?: NewFeatureSupportLevelMap is undefined in some cases.

This part is used in newClustersPage because when we change the Openshift dropdown and CPU architecture dropdown the supportLevelData is altered. 
This implementation was done in version 2.18 (April) and it's working well until now.